tests/test_backward_compat.py

```diff
@@ -17,11 +17,7 @@
 from .generate_old_pickles import PICKLE_DIRECTORY
 
 
-def load_obj(filename, check_deprecation_warning='auto'):
-    if check_deprecation_warning == 'auto':
-        # pickles files generated with cloudpickle_fast.py on old versions of
-        # coudpickle with Python < 3.8 use non-deprecated reconstructors.
-        check_deprecation_warning = (sys.version_info < (3, 8))
+def load_obj(filename, check_deprecation_warning=False):
```

Comment:
Since `_make_skel_func` has been removed we no longer raise a warning in most cases for the tests in this module. Because of this I decided to change `check_deprecation_warning` to be `False` by default, so checking deprecation warnings will need to be done explicitly. Happy to try something else out though